MantidQt/CustomInterfaces/src/Reflectometry/GenericDataProcessorPresenter.cpp

```diff
@@ -1195,7 +1195,7 @@ void GenericDataProcessorPresenter::clearSelected() {
       m_model->setData(m_model->index(*row, i), "");
     }
     // 'Group' column
-    m_model->setData(m_model->index(*row, m_columns - 2), getUnusedGroup());
+    m_model->setData(m_model->index(*row, m_columns - 2), getUnusedGroup(ignore));
```

Comment:
As this was an issue in the generic stuff, would there be a way to regression-test this?